ui-service/src/main/frontend/hotel-booking-ui/src/components/navigation_bar/dropdowns/NavBarSignInDropdown.tsx

```diff
@@ -0,0 +1,52 @@
+import DropdownButton from "./DropdownButton";
+
+export default function NavBarSignInDropdown() {
+  return (
+    <div className="dropdown" style={{ width: "358px" }}>
+      <div className="vip-header">
+        <div>
+          <div
+            className="dropdown-header vip-box"
+            style={{
+              padding: 0,
+              color: "#003c8a",
+            }}
+          >
+            Sign in, or join for free
+          </div>
+          <div className="vip-box" style={{ fontSize: 14 }}>
+            Unlock savings, loyalty perks, and more when you become a VIP.
+          </div>
+        </div>
+        <div>
+          <img src="/join-vip-badge.svg" alt="icon" />
+        </div>
+      </div>
+
+      <DropdownButton text="Sign In" backGroundColor="#0068ef" textColor="white" />
+      <DropdownButton
+        text="Create Account"
+        backGroundColor="#edf0f3"
+        textColor="#0068ef"
+      />
+      <ul className="dropdown-list">
+        <li>
+          <img src="/vip-dark.svg" alt="icon" />
```

Comment:
Why do we save the icon svg in public folder and use img + src here, instead of directly putting the `<svg>...</svg>` here? When in low speed internet, there might be a flashing effect or layout shift.

Reply:
Using img for simplicity reason since svg definition is too long. Not planning to put it in code since the file is so small in size (the largest one is only 9KB) that I won't worry about even so a slow internet.